nva-datamodel-java/src/main/java/no/unit/nva/model/instancetypes/artistic/film/MovingPictureSubtypeEnum.java

```diff
@@ -17,6 +18,7 @@
         this.type = type;
     }
 
+    @Override
```

Comment:
I am not sure whether you want the subtype to be an object or a String. Now it is like this:
```
"publicationInstance" : {
        "type" : "MovingPicture",
        "subtype" : {
          "type" : "InteractiveFilm"
        },
```

If you want it to be a String i.e.,
``` 
    "publicationInstance" : {
        "type" : "MovingPicture",
        "subtype" : "InteractiveFilm"
```

then you have to add a structure similar to the following:
```
@JsonCreator
    public static MovingPictureSubtype fromString(String type){
        
    }
    
    @JsonValue
    public String toString(){
        this.getType();
    }
``` 
and remove the annotation from the field